tests/blackbox/stratisd_cert.py

```diff
@@ -136,39 +137,107 @@ def _unittest_command(self, result, expected_return_code):
 
         self.assertEqual(return_code, expected_return_code, msg=msg)
 
+    def _test_permissions(self, dbus_method, permissions, *args):
+        """
+        Test running dbus_method with and without root permissions.
+        :param dbus_method: StratisDbus wrapper method
+        :param bool permissions: True if dbus_method needs root permissions to succeed.
```

Comment:
Do *args need to be listed here as well?

Reply:
I'ld rather it were just ```args```, and that the arguments to the methods were passed as a list, rather than as a sequence of arguments to the method.

So,

```
:param args: the arguments to be passed to the D-Bus method
:type args: list of object
```

Example invocation:

```
self._test_permissions(
            StratisDbus.pool_add_data, True, [pool_path, StratisCertify.DISKS[2:3]]
        )
```

It's actually less confusing where this function is called.